addons/account_analytic_default_purchase/models/purchase_order_line.py

```diff
@@ -7,15 +7,18 @@
 class PurchaseOrderLine(models.Model):
     _inherit = "purchase.order.line"
 
-    @api.onchange('product_id', 'date_order')
-    def _onchange_product_id_date(self):
-        default_analytic_account = self.env['account.analytic.default'].sudo().account_get(
-            product_id=self.product_id.id,
-            partner_id=self.order_id.partner_id.id,
-            user_id=self.env.uid,
-            date=self.date_order,
-            company_id=self.company_id.id,
-        )
-        if default_analytic_account:
-            self.account_analytic_id = default_analytic_account.analytic_id.id
-            self.analytic_tag_ids = [(6, 0, default_analytic_account.analytic_tag_ids.ids)]
+    account_analytic_id = fields.Many2one(compute='_compute_analytic_id_and_tag_ids', store=True)
```

Comment:
That's not allow in stable. Or did I miss something about this module?

Reply:
@amoyaux It's juste an overwrite of existing fields. So it's ok in stable